{{cookiecutter.repo_name}}/src/util/useHash.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+
+const useHash = (initialValue = null) => {
```

Comment:
Can we leave a note here as to what this does? This is one of those utilities I will def forget about and not use and build it themselves